repository/Javascript-Core.package/GRCountingStream.extension/instance/javascript..st

```diff
@@ -0,0 +1,3 @@
+*javascript-core
+javascript: anObject
+  anObject javascriptOn: self
```

Comment:
This is necessary as GRCountingStream does not extend Stream (which would have the method).